examples/openapi3/jwt/app.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+'''
+Basic example of a resource server
+'''
+
+import time
+
+import connexion
+from werkzeug.exceptions import Unauthorized
+
+from jose import JWTError, jwt
+
+JWT_ISSUER = 'com.zalando.connexion'
+JWT_SECRET = '872f65270a71ca913766335276fb2c1b76b8ccdef749cdfc9a1598ac36d77c99'
```

Comment:
Can you make this something like `not_really_secret`? I can imagine some poor security engineer thinking these are leaked creds.